packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx

```diff
@@ -0,0 +1,179 @@
+  const navigationData = await getNavigationData();
+  return {
+    props: {
+      navigationData,
+    },
+  };
+};
+
+<ArticleHeader
+  title={meta.title}
+  description={meta.description}
+  date={meta.date}
+  machineDate={meta.date}
+  author={meta.author}
+  avatar={meta.avatar}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} hideFeedback />
+
+<ArticleContent>
+
+## 📣 What was new in Paste?
+
+### Filter pattern
+
+<Box marginBottom="space60">
+  <Box
+    borderColor="colorBorderWeaker"
+    borderStyle="solid"
+    borderWidth="borderWidth10"
+    borderRadius="borderRadius20"
+    overflow="hidden"
+  >
+    <Box
+      as={Image}
+      maxWidth="100%"
+      height="100%"
+      src={FiltersPattern}
+      alt="Gif of Filters Pattern"
+      style={{ objectFit: "contain" }}
+    />
+  </Box>
+</Box>
+
+We've updated our [filter pattern](/patterns/filter)! The simplified layout uses Form Pills, Popovers, and Side Panel to reduce visual noise and help customers focus on the data and information that matter most.
+
+### Corner Ornament component
+
+<Box width="100%" maxWidth="400px" display="block" margin="0 auto">
+  <ResponsiveBorderedImage src={CornerOrnament} alt="Preview of Corner Ornament component" />
+</Box>
+
+The [Corner Ornament](/components/corner-ornament) component lets you quickly relate 2 graphic elements to each other. Use it to show unread notifications, cart counts, relationships in Tables, and more. The component supports a variety of Avatar and Icon sizes with built-in cutouts.
+
+### Paste Figma library featuring typography and effect variables
+
+<Box marginBottom="space60">
+  <Box
+    borderColor="colorBorderWeaker"
+    borderStyle="solid"
+    borderWidth="borderWidth10"
+    borderRadius="borderRadius20"
+    overflow="hidden"
+  >
+    <Box
+      as={Image}
+      maxWidth="100%"
+      height="100%"
+      src={FigmaTypography}
+      alt="Gif of Figma Typography"
+      style={{ objectFit: "contain" }}
+    />
+  </Box>
+</Box>
+
+We added typography and effect variables to the [Paste Components & Themes](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/Paste-Components-%26-Themes?type=design&node-id=3015-46646&mode=design&t=LavBWI8WtuCpWbz0-0) Figma library for more accurate swapping between Twilio, Twilio Dark, Evergreen, Default, and Dark themes.
+
+[Learn more about how to swap modes](https://help.figma.com/hc/en-us/articles/15343816063383-Modes-for-variables#h_01H3ADKDF7JBTRV1Z5P1X1DZVE).
+
+### Additional updates
+
+#### New
+
+- Added new [icons](/components/icon) for RCSCapable and SupportRequest.
+
+#### Updates
+
+- [Badge](/components/badge#counter) has a new notification_counter variant.
+
+And thank you for all these contributions, Joe Fehrman!:
+
+- [Form Pill Groups](/components/form-pill-group#large) now have a large variant to support the Filter pattern
+- [User Dialog](/components/user-dialog) has a smaller default size based on feedback from our consumers
```

Comment:
these are missing period at the end.

Reply:
Great catch, they were missing periods in the [spec](https://docs.google.com/document/d/1DrrGpV0BZ6A5UEaCGzSeBgF2o_yTpyTFhupNFz_2LVM/edit) too so I left them, off. Not sure grammatically which is correct so leaving off for now

Reply:
whoops, @PixeledCode is right here. Full sentences in lists should have periods: https://paste.twilio.design/foundations/content/product-style-guide#lists

Good catch!